docs/android/v2/how-to-guides/interact-with-room/peer/large-room.mdx

```diff
@@ -134,3 +134,34 @@ You use **totalCount** property on **HMSPeerListIterator** to know how many peer
 ```kotlin
 let totalPeerCount = peerListIterator.totalCount
 ```
+
+### Search for peers
+To just search for peers without getting all of them use:
+```kotlin
+HMSSDK.searchPeerNameInLargeRoomsearchPeerNameInLargeRoom(
+    query: String,
+    limit: Int,
+    offset: Long,
+    listener: HmsTypedActionResultListener<PeerSearchResponse>) {
+```
+* query - substring of the peer's name that you want to search for.
+* limit - the number of peers you want returned for this query, max <<<>>>>>
```

Comment:
<img width="1223" alt="image" src="https://github.com/user-attachments/assets/2dbd1981-8db0-4d02-b1f3-186c4e65e13a">

Error is due to the angled brackets. Please replace it with the actual value